src/ScrollToTop.tsx

```diff
@@ -2,17 +2,27 @@ import React, { useState, useEffect } from "react";
 
 type Props = {
   top?: number;
+  className?: string;
```

Comment:
Isn't this is included from the component type?  `React.HTMLAttributes<HTMLButtonElement>`

Reply:
@HermanNygaard Ah, maybe. I don't know TypeScript to be honest, looked suspicious to me :D I'll remove this then!